src/lib_json/json_writer.cpp

```diff
@@ -422,8 +422,8 @@ void FastWriter::writeValue(const Value& value) {
       const String& name = *it;
       if (it != members.begin())
         document_ += ',';
-      document_ += valueToQuotedStringN(name.data(),
-                                        static_cast<unsigned>(name.length()));
+      document_ += 
+          valueToQuotedStringN(name.data(), static_cast<size_t>(name.length()));
```

Comment:
remove cast

Reply:
@BillyDonahue, fix it.
https://github.com/open-source-parsers/jsoncpp/pull/1257/commits/cd46649236d7e8e15abfb6c7363f1298a95a0c8b